app/mapping/registration/IndividualBeneficiaryMapper.scala

```diff
@@ -40,7 +40,10 @@ class IndividualBeneficiaryMapper @Inject()(addressMapper: AddressMapper) extend
               beneficiaryType = indBen.roleInCompany.map(_.toString),
               beneficiaryDiscretion = indBen.incomeYesNo,
               beneficiaryShareOfIncome = indBen.income.map(_.toString),
-              identification = identificationMap(indBen)
+              identification = identificationMap(indBen),
+              countryOfResidence = indBen.countryOfResidence,
+              nationality = indBen.countryOfNationality,
+              legallyIncapable = indBen.legallyIncapableYesNo
```

Comment:
Could be worth changing all the references to `legallyIncapableYesNo` to `mentalCapacityYesNo`. Then this line should read `legallyIncapable = indBen.mentalCapacityYesNo.map(!_)` or something like that

Reply:
Updated